templates/ts/m365-message-extension/teamsapp.local.yml.tpl

```diff
@@ -25,6 +25,14 @@ provision:
       # The Azure Active Directory application's client secret created for bot.
       botPassword: SECRET_BOT_PASSWORD 
 
+  # Generate runtime appsettings to JSON file
+  - uses: file/createOrUpdateJsonFile
+    with:
+      target: ./appsettings.Development.json
```

Comment:
It seems that the issue may be related to recent template changes in the development environment. Additionally, Git may encounter problems when handling file renames or moves. @hund030 , do you have any ideas on how to check for template changes?

Reply:
Resolved. Thank you for the finding.

Reply:
Git blame shows that the change is from https://github.com/OfficeDev/TeamsFx/pull/8904, while the PR did not introduce the changes in these templates. I think it's because Git cannot correctly handle file moves.